src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -132,6 +139,127 @@ public function getCrossingChar()
         return $this->crossingChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string      $cross       Crossing char (see #0 of example)
+     * @param string|null $topLeft     Top left char (see #1 of example), equal to $cross if undefined
```

Comment:
`if null`. there is no `undefined` in PHP (E_TOO_MUCH_JAVASCRIPT 😄  ?)

Reply:
Done.